src/components/Section.js

```diff
@@ -0,0 +1,18 @@
+export default class Section {
+  constructor({ items = [], renderer }, cardElement) {
+    this._items = items;
+    this._renderer = renderer;
+    this._cardSelector = document.querySelector(cardElement);
```

Comment:
`_cardSelector`


You haven't corrected it to `_cardsContainer` or just `_container`

Reply:
sorry i thought the cardSelector is the wrong one 
i fixed it